### PR TITLE
add logic to distinguish if form is 'aanvraagStep'

### PR DESCRIPTION
**Pull request by aatauil**
See DL-2814 for more info

### CODE REVIEW

File: addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js [hunk 1]
```diff
@@ -97,6 +115,24 @@ export default class CustomSubsidyFormFieldsEstimatedCostTableEditComponent exte
                                           this.storeOptions.sourceGraph).length > 0;
   }
 
+  get isAanvraagStep() {
+    const results = this.storeOptions.store.match(
+      undefined,
+      optionsPredicate,
+      undefined,
+      undefined);
```
Comment (claire-lovisa) [inline addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js:123]: We can use `this.args.field.options` instead (as discovered while chatting ;))

**Review by claire-lovisa — CHANGES_REQUESTED** [review]
A small change with `this.args.field.options` that'll simplify the code, other than that seems perfect !

File: addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js (hunk 1 above)
Comment (aatauil) [reply addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js]: Changed it to use this.args, also testing if this.args.field && this.args.field.options exist just to be sure

**Review by claire-lovisa — APPROVED** [review]
All good thanks :D

**Pull request merged**